src/Framework/ChangeWaves.cs

```diff
@@ -30,7 +30,8 @@ internal static class ChangeWaves
         internal static readonly Version Wave17_10 = new Version(17, 10);
         internal static readonly Version Wave17_12 = new Version(17, 12);
         internal static readonly Version Wave17_14 = new Version(17, 14);
-        internal static readonly Version[] AllWaves = { Wave17_10, Wave17_12, Wave17_14 };
+        internal static readonly Version Wave17_16 = new Version(17, 16);
```

Comment:
Rename the new changewave to `Wave18_0` at `new Version(18, 0)` and update all uses.